RecoCTPPS/PixelLocal/python/CTPPSPixelClusterizer_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+clusterProd = cms.EDProducer("CTPPSPixelClusterProducer",
```

Comment:
Since changes are applied, I would apply the same convention as for the other configs in CTPPS/Totem reco and call it
  ctppsPixelClusterProducer
unless there is any reason for not using that name....